File: milesToKilometers.java
import java.util.Scanner;

public class milesToKilometers {
	
	public static void main(String [] args) {
		Scanner s = new Scanner(System.in);
		System.out.printf("Please enter a number of miles: ");
	
		double miles = s.nextDouble();
	
		double kms = miles * 1.609344;
	
		System.out.printf("That is %f kilometers\n", kms);
	}
}

File: milesToKilometers.c
/**
*I Coleman
*icoleman@huskers.unl.edu
*2015/08/31
*Miles to Kilometers program
*/

#include<stdlib.h>
#include<stdio.h>

int main(int argc, char **argv) {
	
	printf("Please enter a number of miles: ");
	
	double miles = 0.0;
	
	scanf("%lf", &miles);
	
	double kms = miles * 1.609344;
	
	printf("That is %f kilometers\n", kms);
	
	return 0;
}